packages/zoe/src/zoeService/installationStorage.js

```diff
@@ -20,12 +19,9 @@ const { Fail, quote: q } = assert;
 
 /**
  * @param {GetBundleCapForID} getBundleCapForID
- * @param {Baggage} [zoeBaggage] optional only so it can be omitted in tests
```

Comment:
+1 to `src` not encoding defaults for tests. maybe worth putting in our coding guidelines.